discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from test_infra.controllers.node_controllers import ssh
+
+
+class Host:
+
+    def __init__(self, name, node_controller, private_ssh_key_path=None, username="core"):
+        self.name = name
+        self.private_ssh_key_path = private_ssh_key_path
+        self.username = username
+        self.node_controller = node_controller
+        self._ips = []
+        self._macs = []
+
+    @property
+    def is_active(self):
+        return self.node_controller.is_active(self.name)
+
+    def _set_ips_and_macs(self):
+        self._ips, self._macs = self.node_controller.get_node_ips_and_macs(self.name)
+
+    # TODO maybe add ttl? need mechanism that
+    #  will zero this value when node is stopped
+    @property
+    def ips(self):
+        if not self._ips:
+            self._set_ips_and_macs()
+        return self._ips
+
+    @property
+    def macs(self):
+        if not self._macs:
+            self._set_ips_and_macs()
+        return self._macs
+
+    def run_command(self, bash_command, background=False):
+        output = ""
+        if not self.node_controller.is_active:
+            raise Exception("%s is not active, can't run given command")
+        with ssh.SshConnection(self.ips[0], private_ssh_key_path=self.private_ssh_key_path,
+                               username=self.username) as run:
+            if background:
+                run.background_script(bash_command)
+            else:
+                output = run.script(bash_command, verbose=False)
+        return output
+
+    def shutoff(self):
+        return self.node_controller.shutdown_node(self.name)
+
+    def start_node(self):
```

Comment:
No need for `node` in the function names,  change to `start` and `restart` accordingly